copilot/__init__.py

```diff
@@ -0,0 +1,116 @@
+import ast
+import sys
+import http.client
+import json
+from importlib._bootstrap import spec_from_loader
+
+
+def get_suggestion(file, snippet, stops):
+    conn = http.client.HTTPSConnection("copilot.githubassets.com")
+    payload = json.dumps({
+        "prompt": "// Language: python\n// Path: " + file + "\n" + snippet,
+        "max_tokens": 200,
+        "temperature": 0.2,
+        "top_p": 1,
+        "n": 1,
+        "logprobs": 2,
+        "stop": stops,
+        "stream": False
+    })
+    headers = {
+        'OpenAI-Intent': 'copilot-ghost',
+        'OpenAI-Organization': 'github-copilot',
+        'Authorization': 'Bearer TOKEN',
```

Comment:
It would be a good idea to make TOKEN a constant or parameter even if it's just TOKEN before you get a real token.